content/en/blogs/cilium-as-cni.md

```diff
@@ -0,0 +1,230 @@
+---
+title: 'Cilium as the CNI: Deploy a Kubernetes Cluster with KubeSphere Installed'
```

Comment:
We don't need to emphasize `Cilium as the CNI`. Cilium is relatively new in China but it is popular enough outside of China. Can we update the title? 

My proposal: `Install Cilium on Kubernetes using KubeKey and KubeSphere`.

Reply:
Got your point, but I think `... using KubeKey and KubeSphere` is ambiguous. How about deleting `and KubeSphere`?